browser-test/src/support/admin_questions.ts

```diff
@@ -1215,10 +1222,6 @@ export class AdminQuestions {
     await this.clickSubmitButtonAndNavigate('Create')
 
     await this.expectAdminQuestionsPageWithCreateSuccessToast()
-
-    expectedQuestionText = expectedQuestionText ?? questionText
-
-    await this.expectDraftQuestionExist(questionName, expectedQuestionText)
```

Comment:
suggestion (blocking): We still want to keep `this.expectDraftQuestionExist` for questions that don't include markdown. I just meant to remove the `expectedQuestionText` logic. So this would become:

```
if (!markdown) {
await this.expectDraftQuestionExist(questionName, questionText)
}
```